src/test/java/com/cruru/club/service/ClubServiceTest.java

```diff
@@ -0,0 +1,72 @@
+package com.cruru.club.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.cruru.applicant.domain.repository.ApplicantRepository;
+import com.cruru.club.controller.dto.ClubCreateRequest;
+import com.cruru.club.domain.Club;
+import com.cruru.club.domain.repository.ClubRepository;
+import com.cruru.club.domain.service.ClubService;
+import com.cruru.dashboard.domain.repository.DashboardRepository;
+import com.cruru.member.domain.Member;
+import com.cruru.member.domain.repository.MemberRepository;
+import com.cruru.process.domain.repository.ProcessRepository;
+import java.util.Optional;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@DisplayName("동아리 서비스 테스트")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+class ClubServiceTest {
+
+    @Autowired
+    private MemberRepository memberRepository;
+
+    @Autowired
+    private ClubRepository clubRepository;
+
+    @Autowired
+    private ClubService clubService;
+
+    private Member member;
+
+    @Autowired
+    private ProcessRepository processRepository;
+
+    @Autowired
+    private ApplicantRepository applicantRepository;
+
+    @Autowired
+    private DashboardRepository dashboardRepository;
```

Comment:
별거 아니지만, Repository끼리 모아서 배치해 보는 건 어떨까요?
아래는 예시입니다. 
```suggestion
    @Autowired
    private MemberRepository memberRepository;

    @Autowired
    private ClubRepository clubRepository;
    
    @Autowired
    private ProcessRepository processRepository;

    @Autowired
    private ApplicantRepository applicantRepository;

    @Autowired
    private DashboardRepository dashboardRepository;

    @Autowired
    private ClubService clubService;
    
    private Member member;

```